src/exasol/schemas/templates/index.jinja

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel=stylesheet type=text/css href="https://fonts.googleapis.com/css?family=Overpass:300,400,600,800">
```

Comment:
Would it be possible to include the files locally instead of allowing Google to track our users?

Reply:
I did not see an option for that, added it here to match the css of the generated references ...

Reply:
Argh... I would prefer to remove remote stylesheets, even if it breaks the design.

Reply:
What we can do we can remove it from the overview page, the difference in the font is not to bad. Still removing it from the generated references aint be that straight forward.

Reply:
Updated

Reply:
A closer look revealed, that I have been wrong ... update it. Should be self-contained now.
Good point! :+1: